java/src/jmri/jmrit/logix/Engineer.java

```diff
@@ -618,8 +615,9 @@ protected void setWaitforClear(boolean wait) {
     }
 
     String debugInfo() {
-        StringBuffer info = new StringBuffer("\nEngineer ");
-        info.append(getName()); info.append(", Thread.State= "); info.append(getState());
+        StringBuffer info = new StringBuffer("\n");
+        info.append(getName()); info.append("on warrant= "); info.append(_warrant.getDisplayName());
```

Comment:
Just checking. Does `getName()` return a string that ends with a space character? If not, it seems good to add a space at the beginning of the string `"on warrant= "`.